contracts/test_argent_account_signatures.cairo

```diff
@@ -1,5 +1,5 @@
-use array::ArrayTrait;
 use contracts::ArgentAccount;
+use array::ArrayTrait;
```

Comment:
Best import order from top to bottom is: first most standard/common then libraries then local/custom.

So no need to change this.